lib/routes/watasuke/blog.js

```diff
@@ -0,0 +1,44 @@
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import cheerio from 'cheerio';
+
+const baseUrl = 'https://watasuke.net';
+
+export default async (ctx) => {
+    const blogPageHtml = await got(`${baseUrl}/blog/`).text();
+    const $ = cheerio.load(blogPageHtml);
+    const ogTitle = $('meta[property="og:title"]').attr('content');
+
+    const apiUrl = `${baseUrl}/page-data/blog/page-data.json`;
+    const response = await got(apiUrl);
+    const articles = response.data.result.data.allArticles.nodes;
+
+    const itemsPromises = articles.map(async (article) => {
+        const articleUrl = `${baseUrl}/blog/article/${article.slug}`;
+        const articleHtml = await got(articleUrl).text();
+        const $article = cheerio.load(articleHtml);
+        const ogDescription = $article('meta[property="og:description"]').attr('content') || '説明が見つかりません';
+        const ogImage = $article('meta[property="og:image"]').attr('content') || 'https://watasuke.net/static/b549c082da1248181185ec5d6ffe0a69/thumbnail.jpg';
+
+        return {
+            title: article.title,
+            description: ogDescription,
+            pubDate: parseDate(article.published_at),
+            updated: parseDate(article.updated_at),
+            link: articleUrl,
+            category: article.tags.map((tag) => tag.name),
+            itunes_item_image: ogImage,
+        };
+    });
+
+    const items = await Promise.all(itemsPromises);
+
+    ctx.set('data', {
+        title: ogTitle || String(baseUrl),
```

Comment:
https://github.com/DIYgod/RSSHub/blob/9ea5161178385d289cf29458fe8f645fd8352460/lib/routes/watasuke/blog.js#L5

is already a string. No need to type cast it again.